src/platforms/hosted/platform.c

```diff
@@ -483,6 +483,13 @@ void platform_buffer_flush(void)
 	}
 }
 
+void platform_pace_poll(void)
```

Comment:
Please clang-format this. Looks like it's using two-space indentation with extra spaces in brackets.